File: scrimage-core/src/test/scala/com/sksamuel/scrimage/Issue47Test.scala
package com.sksamuel.scrimage

import org.scalatest.{Matchers, WordSpec}

class Issue47Test extends WordSpec with Matchers {

  "loading iphone image" should {
    "detect rotation flag" in {
      val src = Image.fromResource("/issue47.JPG")
      src.height shouldBe 3264
      src.width shouldBe 2448
    }
  }
}

File: scrimage-core/src/test/kotlin/com/sksamuel/scrimage/core/Issue47Test.kt
package com.sksamuel.scrimage.core

import com.sksamuel.scrimage.ImmutableImage
import io.kotest.core.spec.style.WordSpec
import io.kotest.matchers.shouldBe

class Issue47Test : WordSpec({

   "loading iphone image" should {
      "detect rotation flag"  {
         val src = ImmutableImage.fromResource("/issue47.JPG")
         src.height shouldBe 3264
         src.width shouldBe 2448
      }
   }

})